website/addons/github/static/githubFangornConfig.js

```diff
@@ -8,67 +8,118 @@ var URI = require('URIjs');
 var Fangorn = require('js/fangorn');
 var waterbutler = require('js/waterbutler');
 
+// Cross browser key codes for the Command key
+var commandKeys = [224, 17, 91, 93];
 
 function _uploadUrl(item, file) {
     return waterbutler.buildTreeBeardUpload(item, file, {branch: item.data.branch});
 }
 
+// TODO: Refactor, repeating from core function too much
+function _removeEvent (event, items) {
+    var tb = this;
+    function cancelDelete() {
+        tb.modal.dismiss();
+    }
 
-// Define Fangorn Button Actions
-function _fangornActionColumn (item, col){
-    var self = this;
-    var buttons = [];
+    function runDelete (item) {
+        tb.select('.tb-modal-footer .text-danger').html('<i> Deleting...</i>').css('color', 'grey');;
```

Comment:
Instead of doing these manual DOM updates, can this logic go in the view for an item?